packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -389,7 +392,11 @@ class WebpackCLI {
                         }
 
                         if (index === splitKeys.length - 1) {
-                            prevRef[someKey] = val || true;
```

Comment:
@snitin315 bug is here, I don't think you need diff regex, can you verify? 

what's the result with original regex and this removed?

Reply:
<img width="584" alt="Screenshot 2021-04-18 at 3 28 48 PM" src="https://user-images.githubusercontent.com/46647141/115141583-cfe00d00-a05a-11eb-88a0-bb52d3133c74.png">

Reply:
hmm yes it won't help, lemme thing of something else